README.md

```diff
@@ -1 +1,4 @@
-# rust-symcrypt-sys
+this repo holds two crates; symcrypt (friendly rust wrappers), and symcrypt-sys (FFIs)

```

Comment:
>symcrypt-sys [](http://example.com/codeflow?start=67&length=12)

I think this name could be confusing; to me it kind of implies that this is a kernel driver, which is not the case. Maybe call this `symcrypt-ffi`?

Reply:
-sys is the common naming convention for rust bindings to external C DLLs. For mappings to other windows APIs, see the windows-sys crate.

Reply:
Ah, that's unfortunate because it does seem confusing to me from a Windows perspective, but if that's the Rust convention then I suppose we should follow that.